documentation_builder/io.ipynb

```diff
@@ -35,7 +35,8 @@
     "from cobra.io import load_json_model_model, save_json_model, load_matlab_model, save_matlab_model, read_sbml_model, write_sbml_model\n",
     "import logging\n",
     "\n",
-    "data_dir = pathlib.Path(\"\").joinpath('..', 'tests', 'data').resolve()\n",
+    "data_dir = Path(\".\") / \"..\" / \"src\" / \"cobra\" / \"data\"\n",
+    "data_dir_resolved = data_dir.resolve()\n",
```

Comment:
I don't remember if this is being used anywhere. I wrote it in the beginning of the review, so if it's not being used, you can remove it. My bad.

Reply:
We need to resolve the dir othewise it might now work. Fixed it now.